runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
I thought the point was start does not have to be called.. thus we only have this MUST when start is called. Otherwise this MUST would be over in the process.arg part of the spec. Alternatively could move this MUST to the process.args section and say when start is called...